pages/index.tsx

```diff
@@ -150,21 +142,109 @@ export default function Home() {
               setGraphDataFunc={setGraphData}/>
           </div>
       </div>
-      <br/>
-      <div className="fixed bottom-0 left-0 flex h-48 w-full items-end justify-center bg-gradient-to-t from-white via-white dark:from-black dark:via-black lg:static lg:h-auto lg:w-auto lg:bg-none">
-            <CertifyBadSelect 
-              label="CertifyBad Entities" 
-              options={certifyBadEntries} 
-              setGraphDataFunc={setGraphData} />
-      </div>
-      <div>
-        {
-          cjsGraphData.map((d)=> {
-            const [start, graphData] = processDataForCytoscape(d.data);
-            return <Graph key={d.key} layout="dagre" writeDetails={()=>{}} startNode={start} graphData={graphData} /> 
-          })
-        }
-      </div>
+      <div className="grid grid-cols-3">
+        <div className="w-full items-left justify-left font-mono text-sm p-24 lg:col-span-1">
+          {/* <h2>Highlight Nodes</h2>
```

Comment:
Why commented out? Do you still plan to work on it? If so can you just add a todo?